### PR TITLE
Updated environment_yml

### PR DESCRIPTION
**Pull request by WojciechowskiDaniel**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -13,3 +13,7 @@ dependencies:
   # TODO nf-core: Add required software dependencies here
   - bioconda::fastqc=0.11.8
   - bioconda::multiqc=1.7
```
Comment (twrzes) [inline environment.yml]:
```suggestion
```
As neither FastQC, nor MultiQC will be used in `sim-seqer` tool, I think you can safely remove those dependencies from `environment.yml` file

**Review by twrzes — CHANGES_REQUESTED** [review]
Looks fantastic!

File: environment.yml (hunk 1 above)
Comment (WojciechowskiDaniel) [reply environment.yml]: Removed

**Review by twrzes — APPROVED** [review]
Great change, I appreciate your effort!

**Pull request merged**